package.json

```diff
@@ -1,56 +1,77 @@
+        "default": "./build/cjs/index.js"
+      },
+      "default": {
+        "types": "./build/esm/index.d.js",
+        "style": "./build/index.css",
+        "default": "./build/esm/index.js"
+      }
+    },
+    "./styles": "./build/index.css",
+    "./i18n/*": "./src/i18n/*",
+    "./themes/*": {
+      "style": "./build/themes/*/index.css",
+      "default": "./build/themes/*/index.css"
+    }
+  },
   "sideEffects": [
-    "./build/index.js",
-    "./build/i18n/index.js",
-    "./src/index.ts",
-    "./src/i18n/index.ts",
+    "*.css",
     "*.scss"
   ],
-  "main": "build/index.js",
-  "typings": "build/index.d.ts",
+  "main": "build/esm/index.js",
+  "typings": "build/esm/index.d.ts",
   "scripts": {
     "deps:install": "npm ci",
     "deps:truncate": "npm prune --production",
-    "lint": "run-p lint:js lint:styles lint:prettier",
-    "lint:fix": "run-s lint:js:fix lint:styles:fix lint:prettier:fix",
-    "lint:js": "eslint 'src/**/*.{js,jsx,ts,tsx}'",
-    "lint:js:fix": "eslint 'src/**/*.{js,jsx,ts,tsx}' --quiet --fix",
-    "lint:styles": "stylelint src/**/*.scss",
-    "lint:styles:fix": "stylelint **/*.scss --fix",
-    "lint:prettier": "prettier --check 'src/**/*.{js,jsx,ts,tsx,css,scss,json,yaml,yml,md}'",
-    "lint:prettier:fix": "prettier --write 'src/**/*.{js,jsx,ts,tsx,css,scss,json,yaml,yml,md}'",
+    "lint": "run-p _lint:js _lint:styles _lint:prettier",
+    "lint:fix": "run-s _lint:js:fix _lint:styles:fix _lint:prettier:fix",
+    "_lint:js": "eslint '**/*.{js,jsx,ts,tsx}'",
+    "_lint:js:fix": "npm run _lint:js -- --fix",
+    "_lint:prettier": "prettier --check 'src/**/*.{js,jsx,ts,tsx}'",
+    "_lint:prettier:fix": "prettier --write 'src/**/*.{js,jsx,ts,tsx}'",
+    "_lint:styles": "stylelint src/**/*.scss",
+    "_lint:styles:fix": "npm run _lint:styles -- --fix",
     "typecheck": "tsc --noEmit",
-    "svgo": "svgo --config svgo.config.js",
-    "svgo:all": "svgo --config svgo.config.js -r -f .",
-    "dev": "npm run storybook:start",
-    "test": "jest --passWithNoTests",
-    "test:update": "jest -u",
+    "test": "exit 0",
+    "dev": "run-s build _dev:watch",
+    "_dev:watch": "run-p _build:watch _storybook:watch",
     "build:clean": "rimraf build",
-    "build:copy": "copyfiles -u 1 \"src/components/**/*.scss\" \"src/components/**/*.svg\" build",
-    "build:compile": "tsc -p tsconfig.publish.json",
-    "build": "npm run build:clean && npm run build:compile && npm run build:copy",
-    "storybook:start": "start-storybook -p 7008",
-    "storybook:build": "build-storybook -c .storybook -o storybook-static",
+    "build:compile": "./esbuild/build.js",
+    "_build:declarations:esm": "tsc --emitDeclarationOnly -p tsconfig.esm.json",
+    "_build:declarations:cjs": "tsc --emitDeclarationOnly -p tsconfig.cjs.json",
+    "build": "run-s build:clean build:compile _build:declarations:*",
+    "_build:watch": "./esbuild/build.js --watch",
+    "_storybook:watch": "cd demo && npm run storybook",
```

Comment:
there is no "storybook" in package.json

Reply:
you need to run `npm ci` in `demo folder`. maybe one should add "postinstall" script?